Formula/mariadb.rb

```diff
@@ -9,13 +9,11 @@ class Mariadb < Formula
   # requirement. In the future, this will be updated to use a `Json` strategy
   # and we can remove the unused regex at that time.
   livecheck do
-    url "https://downloads.mariadb.org/rest-api/mariadb/all-releases/?olderReleases=false"
+    url "https://downloads.mariadb.org/rest-api/mariadb/10.6/"
```

Comment:
Given this is an unversioned formula, I think it makes sense how it was before where it searched for the latest stable rather than the latest 10.6.

The rest LGTM however.

Reply:
Sounds plausible. Dropped the commit for `mariadb`.